packages/@o3r/rules-engine/src/components/rules-engine/ruleset-history/ruleset-history-pres.template.html

```diff
@@ -7,6 +7,7 @@ <h4 class="mb-4">Ruleset Execution History</h4>
   </ng-template>
   <ul *ngIf="rulesetExecutions; else noRulesEngine" class="rulesets">
     <li *ngFor="let execution of rulesetExecutions" class="ruleset">
+      <!-- eslint-disable-next-line @angular-eslint/template/click-events-have-key-events, @angular-eslint/template/interactive-supports-focus -- need to refactor the div to accordion from DF -->
```

Comment:
we can link the associated issue https://github.com/AmadeusITGroup/otter/issues/1518